src/apps/frontend/constants/routes.ts

```diff
@@ -4,6 +4,8 @@ const routes = {
   FORGOT_PASSWORD: '/forgot-password',
   LOGIN: '/login',
   RESET_PASSWORD: '/accounts/:accountId/reset_password',
+  OTP: '/otp/verify',
+  PHONE_LOGIN: '/login',
```

Comment:
No need to create a new route as the login is same for both email and phone. Best to use `LOGIN` constant